src/js/profile/mobile/widget/Calendar.js

```diff
@@ -43,7 +43,9 @@
 				today = new Date(),
 				todayYear = today.getFullYear(),
 				todayMonth = today.getMonth() + 1,
+				defaultToday = today.getUTCDate(),
 				switchElement,
+				selectDay,
```

Comment:
This property will be common for al instances of Calendar widget. I this will be better assign it to `this`. For example  now, when you change month the selection is moving to the same day selected in previous month.